sql/wsrep_sst.cc

```diff
@@ -610,8 +610,8 @@ static void* sst_joiner_thread (void* a)
     else
     {
       err= proc.error();
-      WSREP_ERROR("Failed to execute: %s : %d (%s)",
-                  arg->cmd, err, strerror(err));
+      WSREP_ERROR("Failed to execute (%M): %s",
```

Comment:
As a part of #3360, I noted that `WSREP_ERROR` does *not* use `my_vsnprintf` and thus will not recognize `%M`:
https://github.com/MariaDB/server/blob/f644b9945b4286d44c64948160a4a7b6a10a13a5/include/wsrep.h#L27-L33
https://github.com/MariaDB/server/blob/f644b9945b4286d44c64948160a4a7b6a10a13a5/sql/wsrep_mysqld.cc#L289-L326